topics/algolia/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: algoliasearch, algolia-search, instantsearch
```

Comment:
Hmm, actually, would you move `instantsearch` to the related field? You'd need a new `related: instantsearch` line. It will have the same effect (showing up in "Related topics"), but the meaning is a bit different. See https://github.com/topics/angular and how `vue` and `react` are related to `angular` in https://github.com/github/explore/blob/d336a4b685f26a73b53068a72fd739a7b4391ea4/topics/angular/index.md, for example.

Reply:
that makes sense, updated!